web/src/views/events/EventView.tsx

```diff
@@ -479,17 +496,19 @@ function DetectionReview({
                 </div>
               );
             })}
-          {(itemsToReview ?? 0) > 0 && (
-            <div className="col-span-full flex justify-center items-center">
-              <Button
-                className="text-white"
-                variant="select"
-                onClick={markAllReviewed}
-              >
-                Mark these items as reviewed
-              </Button>
-            </div>
-          )}
+          {currentItems != null &&
```

Comment:
could this just be `{currentItems?.length > 0 && itemsToReview > 0 && (`? Maybe I'm missing something that you're checking explicitly for?